src/cloud_provider/digitalocean/kubernetes/mod.rs

```diff
@@ -85,8 +85,8 @@ impl<'a> DOKS<'a> {
         let mut context = TeraContext::new();
 
         // VPC segmentation
-        let vpc_name = "qovery";
-        context.insert("vpc_name", vpc_name);
+        let vpc_name = format!("qovery-{}", &self.id);
```

Comment:
Warning, I've already fixed this yesterday. I think you'll have to rebase. Also, I think prefixing is a good idea (I didn't did it) https://github.com/Qovery/engine/pull/60